app/services/form1010cg/service.rb

```diff
@@ -66,7 +78,7 @@ def process_claim!
     # @return [Boolean]
     def submit_attachment # rubocop:disable Metrics/MethodLength,Metrics/CyclomaticComplexity
       raise 'requires a processed submission'     if  submission&.carma_case_id.blank?
-      raise 'submission already has attachments'  if  submission.attachments.any?
+      raise 'submission already has attachments'  if  submission.attachments&.any?
```

Comment:
Does submission.attachments need a safe navigation operator? submission&.attachments?

Reply:
It doesn't since we are checking the submission exists above.

Reply:
Actually, after a second read, we are missing a check for submission. Thanks for pointing that out.

Reply:
Let me know when the changes are ready and I can re-review

Reply:
So fun fact: This **does** check for an empty submission because I chain `#.blank?` at the end and even had a [test case](https://github.com/department-of-veterans-affairs/vets-api/pull/5465/files#diff-2b9c7a4aee6c61300caecf5088fb4776fd710d5626970b0620e2f8be7dd67440R751) for it. But since that control flow is confusing (even for me who wrote it),  I'll add an additional clause to make it more readable.

Reply:
Awesome, thanks for the explanation and looking into this!